ISSUE_GUIDELINES.md

```diff
@@ -0,0 +1,44 @@
+# Issue Guidelines
+
+Issues filed in this repository are specific to the Kubecost Helm chart only. If your issue pertains to the Kubecost application and not the Helm chart, follow the guidance below.
+
+To help route you to the best location, see some examples of feature requests and bug reports [below](#examples-of-helm-requests).
+
+## Kubecost Feature Requests
+
+For all feature requests to Kubecost, please create an enhancement request in the [feature requests repository](https://github.com/kubecost/feature-requests).
```

Comment:
How about Kubecost application bugs/issues? i.e. outside of the helm chart.

Reply:
See the section just below where I spell those out.